src/main/java/de/doubleslash/keeptime/view/SettingsController.java

```diff
@@ -203,10 +238,53 @@ private void initialize() {
       initExportButton();
       initImportButton();
 
+      toggleGroup = new ToggleGroup();
+      radioApiOff.setToggleGroup(toggleGroup);
+      radioApiOn.setToggleGroup(toggleGroup);
+
+      Properties properties = new Properties();

```

Comment:
in the ui
- a hint that changes will only be applied after restarting keeptime 
- a hint that it is a beta feature
would be nice